app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +26,17 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          var noclass = {
+            metadata: {
+              name: "No Storage Class",
+              labels: {},
+              annotations: {
+                description: "No storage class will be assigned unless a default class has been assigned by the system administrator."
+                }
+            }
+          };
```

Comment:
My understanding is the default storage class has an annotation, so we should know if there is one and which one it is. I think we can improve the user experience here. Rather than a dummy "no storage class" option, we might just let users leave the field blank and then say in the help text what the default is (when there is a default).

cc @jwforres